packages/webpack/src/buildtime/aa.js

```diff
@@ -90,6 +92,20 @@ exports.generateIdentifierLookup = ({
         }
       }
     }
+    for (const c of contextModules) {
+      const resourceId = getPackageNameForModulePath(
```

Comment:
nit: why `resourceId`? I mean, `getPackageNameForModulePath` sounds like it returns a name, right? I get that the name is effectively an identifier, but at the same time a bit confusing because id reminds me of something different than a name? I'm stating this just in case there's an explanation of this worth hearing, but either way feel free to ignore

Reply:
we're calling it multiple things interchangeably. canonical name == resource. aa uses canonical name, policy file has a key called resources.
canonical name is a decent internal name, but I'm slowly switching to calling them resources to be consistent with policy.